plugins/woocommerce/includes/wc-template-functions.php

```diff
@@ -3336,7 +3336,7 @@ function wc_display_item_meta( $item, $args = array() ) {
 			)
 		);
 
-		foreach ( $item->get_formatted_meta_data() as $meta_id => $meta ) {
+		foreach ( $item->get_all_formatted_meta_data() as $meta_id => $meta ) {
```

Comment:
IMO we should not display all meta here, since this function is generally used in frontend template. wdyt?

Reply:
Hey thanks for the review @vedanshujain - I changed this because on the order summary page this is how it displays:
| `get_all_formatted_meta_data` | `get_formatted_meta_data` |
|---|---|
| ![image](https://user-images.githubusercontent.com/5656702/146181483-e20779e1-0681-46ad-99a8-09c1aa0933b2.png) | ![image](https://user-images.githubusercontent.com/5656702/146181535-24edb423-ab7b-4b19-88d3-9385d233e64e.png) |

I think we need to keep it as `get_all_formatted_meta_data` here to solve the issue.